Contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
```

Comment:
Maybe we better to create a CHANGELOG and ask the contributor to update that file and of course update the README only when relevant

Reply:
that's sounds better, lets see what others think

Reply:
Changes to the README should probably exist only when there are changes relevant to the essence of the project, or to the ways to deploy it.